WINDOW_CHECK/Window_Check.py

```diff
@@ -0,0 +1,204 @@
+from collections import Counter
+#from collections import OrderedDict
+import numpy as np
+import os
+import re
+import subprocess
+import sys
+
+exec(open('/data/acis/LoadReviews/script/WINDOW_CHECK/SI_Mode_Class.py').read())
```

Comment:
Any chance we can import the class from this file instead? If they are in the same directory it should work.

Reply:
Agreed this really should be fixed.  Using `exec` in this way is not acceptable for production code as it completely bypasses the important concept of namespace encapsulation.  This is equivalent to `from module import *` which injects a completely unknown set of symbols into this module namespace.  That means you cannot know where a symbol comes from nor get useful results from automated code linters.

Using exec also adds the additional downside that the hard-coded absolute path makes it difficult or impossible to install and test an isolated development version of the suite of `lr` tools.

Reply:
This was fixed and pushed up days ago....March 2 to be exact.

The intention was not to keep the exec.

Reply:
Perfect.  I missed the Outdated tag, my fault.  It does help outside reviewers to mark a conversation as resolved  ("Resolve conversation" below) or reply "Fixed" or "Done" when you fix it.

Reply:
I shall endeavor to reply "Fixed" or "Dome" in the future.